kie-archetypes/kie-kjar-archetype/src/main/resources/WorkDefinitions.wid

```diff
@@ -0,0 +1,89 @@
+[
+  [
+    "name" : "Email",
+    "parameters" : [
+      "From" : new StringDataType(),
+      "To" : new StringDataType(),
+      "Subject" : new StringDataType(),
+      "Body" : new StringDataType()
+    ],
+    "displayName" : "Email",
+    "icon" : "defaultemailicon.gif"
+  ],
+
+  [
+    "name" : "Log",
+    "parameters" : [
+      "Message" : new StringDataType()
+    ],
+    "displayName" : "Log",
+    "icon" : "defaultlogicon.gif"
+  ],
+
+  [
+    "name" : "WebService",
+    "parameters" : [
+        "Url" : new StringDataType(),
+         "Namespace" : new StringDataType(),
+         "Interface" : new StringDataType(),
+         "Operation" : new StringDataType(),
+         "Parameter" : new StringDataType(),
+         "Endpoint" : new StringDataType(),
+         "Mode" : new StringDataType()
+    ],
+    "results" : [
+        "Result" : new ObjectDataType(),
+    ],
+    "displayName" : "WS",
+    "icon" : "defaultservicenodeicon.png"
+  ],
+
+  [
+    "name" : "Rest",
+    "parameters" : [
+        "ContentData" : new StringDataType(),
+        "Url" : new StringDataType(),
+        "Method" : new StringDataType(),
+        "ConnectTimeout" : new StringDataType(),
+        "ReadTimeout" : new StringDataType(),
+        "Username" : new StringDataType(),
+        "Password" : new StringDataType()
+    ],
+    "results" : [
+        "Result" : new ObjectDataType(),
+    ],
+    "displayName" : "REST",
+    "icon" : "defaultservicenodeicon.png"
+  ],
+
+  [
+    "name" : "Milestone",
+    "parameters" : [
+      "Condition" : new StringDataType()
+    ],
+    "displayName" : "Milestone",
+    "icon" : "defaultmilestoneicon.png",
+    "category" : "Milestone"
+  ],
+
+  [
+    "name" : "StartCaseInstance",
+    "parameters" : [
+      "DeploymentId" : new StringDataType(),
+      "CaseDefinitionId" : new StringDataType(),
+      "Data_" : new StringDataType(),
+      "UserRole_" : new StringDataType(),
+      "GroupRole_" : new StringDataType(),
+      "DataAccess_" : new StringDataType(),
+      "Independent" : new StringDataType(),
+      "DestroyOnAbort" : new StringDataType()
+    ],
+    "results" : [
+        "CaseId" : new StringDataType(),
+    ],
+    "displayName" : "Sub Case",
+    "icon" : "defaultsubcaseicon.png",
+    "category" : "Cases"
+  ]
```

Comment:
maybe add decision and business rule task work items here, wdyt?

Reply:
added

Reply:
Just to clarify - what is this file for? Is it included in the generated kjar? It is not under archetype-resources so probably it is not?

Reply:
It will be included in the generated kjar under /src/main/resources

Reply:
So what is the difference between this file called workdefinitions.wid and the file workdefinition.wid which is in src/main/resources/archetype-resources/src/main/resources/WorkDefinition.wid?

Reply:
Opps nice catch! I removed this one. Sorry!